<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>나눔고딕코딩</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>EMS SCADA</string>
  </property>
  <property name="iconSize">
   <size>
    <width>64</width>
    <height>64</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="totalFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1181</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="headFrame">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1181</width>
       <height>80</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(41, 42, 55);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="iconFrame">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>5</y>
        <width>64</width>
        <height>67</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-repeat: no-repeat;
background-image: url(:/iot);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lbltitle">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>13</y>
        <width>370</width>
        <height>52</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>EMS IoT DASHBOARD</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="greetingFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>94</y>
       <width>220</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(60, 60, 90);
border-style:outset;
border-width:5px;
border-color: rgb(0, 0, 255);
border-radius:10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="weatherFrame">
      <property name="geometry">
       <rect>
        <x>79</x>
        <y>170</y>
        <width>64</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/clear);
background-repeat:no-repeat;
border:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblDate">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>20</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border:none;</string>
      </property>
      <property name="text">
       <string>9999-99-99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblDay">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>60</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border:none;</string>
      </property>
      <property name="text">
       <string>Wednesday</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTime">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>104</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 170, 0);
border:none;</string>
      </property>
      <property name="text">
       <string>99:99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblGreeting">
      <property name="geometry">
       <rect>
        <x>6</x>
        <y>250</y>
        <width>210</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(210, 210, 210);
border:none;</string>
      </property>
      <property name="text">
       <string>Good Afternoon!</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QFrame" name="conframe">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/red);
background-repeat:no-repeat;
border:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblstatus">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>410</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(210, 210, 210);
border:none;</string>
      </property>
      <property name="text">
       <string>Disconnected</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="tempFrame">
     <property name="geometry">
      <rect>
       <x>234</x>
       <y>94</y>
       <width>250</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(72, 0, 0);
border-style:outset;
border-width:5px;
border-color: rgb(180, 50, 0);
border-radius:10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="tempIconFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/temp);
background-repeat:no-repeat;
border:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempTitle">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>9</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
      </property>
      <property name="text">
       <string>EMS99 Temperature</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrentTemp">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>170</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 100, 25);
border:none;</string>
      </property>
      <property name="text">
       <string>99.9</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblTempC">
      <property name="geometry">
       <rect>
        <x>178</x>
        <y>180</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 100, 25);
border:none;</string>
      </property>
      <property name="text">
       <string>℃</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="HumidFrame">
     <property name="geometry">
      <rect>
       <x>488</x>
       <y>94</y>
       <width>250</width>
       <height>481</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 50, 115);
border-style:outset;
border-width:5px;
border-color: rgb(0, 50, 180);
border-radius:10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="HumidIconFrame">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>128</width>
        <height>128</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-image: url(:/humid);
background-repeat:no-repeat;
border:none;
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
     <widget class="QLabel" name="lblHumidTitle">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>9</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
      </property>
      <property name="text">
       <string>EMS99 Humidity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblCurrentHumid">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>100</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>48</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color: rgb(108, 191, 248);</string>
      </property>
      <property name="text">
       <string>99</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="lblHumidP">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>180</y>
        <width>70</width>
        <height>70</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
color: rgb(108, 191, 248);</string>
      </property>
      <property name="text">
       <string>％</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="lblframe">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>100</y>
       <width>440</width>
       <height>480</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>나눔고딕코딩</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 50, 115);
border-style:outset;
border-width:5px;
border-color: rgb(0, 50, 180);
border-radius:10px;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lblchart">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>9</y>
        <width>190</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>나눔고딕코딩</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
border:none;</string>
      </property>
      <property name="text">
       <string>visuallization</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="dashboard.qrc"/>
 </resources>
 <connections/>
</ui>
